hunchentoot.asd

```diff
@@ -26,43 +26,37 @@
 ;;; NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 ;;; SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-(in-package :cl-user)
-
-(defpackage :hunchentoot-asd
-  (:use :cl :asdf))
-
-(in-package :hunchentoot-asd)
-
-(defsystem :hunchentoot
+(defsystem "hunchentoot"
```

Comment:
For consistency, can you please use keywords for system names ?

Reply:
I do not mind making this change, but I'd like you to know (and the issue tracker to reflect) why I prefer string names: ASDF components are named by case-sensitive strings internally, and I find the process of coercing symbols into strings ASDF uses wonky. note that `:hunchentoot`, `:HUNCHENTOOT` and `:|hUnChEnToOt|` are all the same system as `"hunchentoot"`, but none of those is the same system as `"HUNCHENTOOT"`.

Also, consistency with what?

Reply:
Consistency with the current style, and the vast majority of Lisp ecosystem.
As for the downcasing, that's fine because almost nobody names systems any other than all-lowcase.